package.json

```diff
@@ -517,17 +517,18 @@
     "chai": "^4.2.0",
     "coffee-script": "^1.10.0",
     "coveralls": "^3.0.2",
+    "cross-env": "^5.2.0",
```

Comment:
I understand the addition of this devDependency. Are the others being updated here related to _this_ PR?

Reply:
I don't think so; didn't intend this.  I'll pull them out.  note to self: don't run `npm upgrade` unless you mean it.

Reply:
Removed from here, but "package-lock.json" not regenerated...

Reply:
it’s the same because the semver ranges install the same packages at this point.

Reply:
at least that’s my theory. I’ll double check that it doesn’t get modified if I install again

Reply:
yep, no change.